src/react/TabsRuntime.ts

```diff
@@ -21,7 +21,7 @@ export function TabsRuntime(props: Partial<TabsControllerOptions> = {}) {
             ...tabs.getTabsFromLocalStorage(),
             ...tabs.getTabsFromSearchQuery(),
         });
-    }, [tabs]);
+    }, [props.router]);
```

Comment:
here is only `props.router`? `tabs` or other `props` are not needed?

Reply:
restore works only when page changed, tabs not changing